{
    "id": "CVE-2024-53908",
    "product": "n/a",
    "version": "4.2 ",
    "vulnerability": [
        "n/a"
    ],
    "description": "An issue was discovered in Django 5.1 before 5.1.4, 5.0 before 5.0.10, and 4.2 before 4.2.17. Direct usage of the django.db.models.fields.json.HasKey lookup, when an Oracle database is used, is subject to SQL injection if untrusted data is used as an lhs value. (Applications that use the jsonfield.has_key lookup via __ are unaffected.)",
    "poc": {
        "reference": [
            "No PoCs from references."
        ],
        "github": [
            "https://github.com/ch4n3-yoon/ch4n3-yoon"
        ]
    }
}